src/modules/api.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] KIndly remove this & solve the linter manually, if the error present.

Reply:
well resolved! ✔️